src/main/java/org/folio/dataimport/handlers/actions/CreateInvoiceEventHandler.java

```diff
@@ -267,7 +270,7 @@ private Future<Map<Integer, PoLine>> getAssociatedPoLinesByRefNumbers(Map<Intege
     return requestContext.getContext()
       .<List<Future<Pair<Integer, PoLine>>>>executeBlocking(promise -> {
         List<Future<Pair<Integer, PoLine>>> futures = new ArrayList<>();
-        Semaphore semaphore = new Semaphore(SEMAPHORE_MAX_ACTIVE_THREADS, Vertx.currentContext().owner());
+        Semaphore semaphore = new Semaphore(maxActiveThreads, Vertx.currentContext().owner());
```

Comment:
@grabsefx should https://github.com/folio-org/mod-invoice/pull/423/files#diff-6918b7006c1e5037f959e6c28c668ab94e2090f975d6c03d9177359051a131acR374 be change as well ?